### PR TITLE
Fix/jira

### PR DESCRIPTION
**Pull request by erlend-vimond**
## *What* does this PR do?
Makes requests to JIRA without using the jira-search module. If one of the tickets are invalid, the jira-search module fails. However, the JIRA API returns which tickets were valid and not, and we can use that to simply filter the tickets rather than failing.

## *Who* is the change affecting?
Eventually everyone

## *What* is affected by this change?
The requests made to JIRA

## JIRA Tickets
- PLAT-648
- PLAT-564

## Manual Testing
It fails because it doesn't find the fetch, node-fetch nor axios modules. Please help.

## Work in Progress
The code only works locally due to the above mentioned issue.

### CODE REVIEW

File: utils/jira.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        maxResults: maxResults
+    }),
+        {
+            headers: {
+                "Content-Type": "application/json",
+                "Authorization": `Basic ${auth}`
+            }
+        });
+
+    if (f.status != 200) {
```
Comment (dagvl) [inline utils/jira.js:27]: `!==` unless you want type coercion to happen

**Review by dagvl — APPROVED** [review]
Two very minor things, otherwise LGTM

**Comment by erlend-vimond** [comment]
Are you sure we should await the top level functions? When trying to build it, I now get this error:
```
❯ npm install      

> javascript-action@1.0.0 prepare
> ncc build index.js -o dist --source-map --license licenses.txt

ncc: Version 0.31.1
ncc: Compiling file index.js into CJS
Error: Module parse failed: Top-level-await is only supported in EcmaScript Modules
File was processed with these loaders:
 * ./node_modules/@vercel/ncc/dist/ncc/loaders/empty-loader.js
 * ./node_modules/@vercel/ncc/dist/ncc/loaders/relocate-loader.js
 * ./node_modules/@vercel/ncc/dist/ncc/loaders/shebang-loader.js
You may need an additional loader to handle the result of these loaders.
Error: Top-level-await is only supported in EcmaScript Modules
    at /Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:697605
    at Hook.eval [as call] (eval at create (/Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:117670), <anonymous>:7:16)
    at Hook.CALL_DELEGATE [as _call] (/Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:114948)
    at JavascriptParser.walkAwaitExpression (/Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:907582)
    at JavascriptParser.walkExpression (/Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:906265)
    at JavascriptParser.walkExpressionStatement (/Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:896473)
    at JavascriptParser.walkStatement (/Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:895354)
    at JavascriptParser.walkStatements (/Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:893245)
    at JavascriptParser.parse (/Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:921381)
    at /Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:423530
    at /Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:1862758
    at /Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:336439
    at _done (eval at create (/Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:117892), <anonymous>:9:1)
    at eval (eval at create (/Users/erlend/projects/actions/ticket-collector/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:117892), <anonymous>:34:22)
npm ERR! code 1
npm ERR! path /Users/erlend/projects/actions/ticket-collector
npm ERR! command failed
npm ERR! command sh -c ncc build index.js -o dist --source-map --license licenses.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/erlend/.npm/_logs/2022-08-30T10_39_31_145Z-debug-0.log
```

**Comment by dagvl** [comment]
> @dagvl Are you sure we should await the top level functions? When trying to build it, I now get this error:

Hm.. my mistake. I was only working in the IDE and it must have compiled it as a EcmaScript module instead of CJS. Javascript is a complicated ecosystem... I'll revert.

**Comment by dagvl** [comment]
If you are done with all the jira changes needed, please merge the PR

**Pull request merged**